core/controllers/admin.py

```
@@ -1531,6 +1568,14 @@ def generate_dummy_story_nodes(
                                 'new_value': exp_id,
                             }
                         ),
+                        story_domain.StoryChange(
+                            {
+                                'cmd': 'move_node_to_arc',
+                                'node_id': '%s%d'
```

Ruff (0.15.15)

[warning] Use format specifiers instead of percent format

(UP031)